python/ct/test/time_utils.py

```diff
@@ -0,0 +1,27 @@
+"""Utilities for testing time-sensitive code."""
+
+import contextlib
+import os
+import time
+
+
+def get_timezone_environ():
+    return os.environ.get("TZ", '')
+
+def set_timezone_environ(new_timezone):
+    os.environ["TZ"] = new_timezone
+    time.tzset()
+
+@contextlib.contextmanager
```

Comment:
Not sure what timezone @linshunghuang is in, but he's reported TZ-related bugs in the cert_desc.py tests a couple of times, so it'd be good to add his TZ to the test case.